plugins/email_service/app/controllers/email_service/stats_controller.rb

```diff
@@ -1,16 +1,12 @@
 module EmailService
   class StatsController < ::EmailService::ApplicationController
     before_action :restrict_access
-
+    before_action :check_ec2_creds_cronus_status
```

Comment:
same here

Reply:
This is not checking roles anymore. It is checking whether the user has EC2 credentials and nebula enabled or not.